vks-ai-conformance/v1.34/gang_scheduling.md

```diff
@@ -0,0 +1,373 @@
+## Description 
+
+The platform must allow for the installation and successful operation of at least one gang scheduling solution that ensures all-or-nothing scheduling for distributed AI workloads (e.g. Kueue, Volcano, etc.) To be conformant, the vendor must demonstrate that their platform can successfully run at least one such solution.
+
+## Evidence
+
+### Prerequisites
+
+* Provision a VKS v3.5.0 Cluster with v1.34.1 node pool, VM Class with vGPU profile and NVIDIA GPU Operator
+
+* Log in to the cluster
+
+References:
+
+- https://techdocs.broadcom.com/us/en/vmware-cis/private-ai/foundation-with-nvidia/9-0/private-ai-foundation-9-x/deploying-ai-workloads-on-tkg-clusters/deploy-a-gpu-accelerated-tkg-cluster-with-kubectl-connected.html
+
+- https://techdocs.broadcom.com/us/en/vmware-cis/vcf/vsphere-supervisor-services-and-standalone-components/latest/managing-vsphere-kuberenetes-service-clusters-and-workloads/deploying-ai-ml-workloads-on-tkg-service-clusters/cluster-operator-workflow-for-deploying-ai-ml-workloads-on-tkg-service-clusters.html
+
+### Install Kueue
+
+```shell
+## Create namespace for kueue install
+kubectl create ns  kueue-system
+
+###Install via helm
```

Comment:
super nit: `###Install via helm` -> ## Install via helm` (not sure why the double `#` but to be consistent with above...)